src/vnsw/agent/vrouter/flow_stats/flow_stats_manager.cc

```diff
@@ -89,6 +89,10 @@ FlowStatsManager::FlowStatsManager(Agent *agent) : agent_(agent),
     flow_export_drops_ = 0;
     flows_sampled_atleast_once_ = false;
     request_queue_.set_measure_busy_time(agent->MeasureQueueDelay());
+    for (uint16_t i = 0; i < sizeof(protocol_list_)/sizeof(protocol_list_[0]);
```

Comment:
Fix for this bug is taken from current master on Juniper's repo:

https://github.com/Juniper/contrail-controller/blob/R4.1/src/vnsw/agent/vrouter/flow_stats/flow_stats_manager.cc#L103